api-reference/10 UI Components/dxFileManager/1 Configuration/onFileUploaded.md

```diff
@@ -5,25 +5,175 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed when a file is successfully uploaded.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.component): dxFileManager
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "component" }
 
 ##### field(e.fileData): File
-<!-- Description goes here -->
+The uploaded file.
```

Comment:
Maybe be better "The file to be uploaded" since this event occurs before upload operation?